mozetl/hardware_report/summarize_json.py

```diff
@@ -600,28 +599,21 @@ def generate_report(start_date, end_date):
 
     # Connect to the longitudinal dataset.
     sqlQuery = """
-               SELECT
-                   build,
-                   client_id,
-                   active_plugins,
-                   system_os,
-                   submission_date,
-                   system,
-                   system_gfx,
-                   system_cpu,
-                   normalized_channel,
+               SELECT 
+                  build, 
+                  client_id, 
+                  active_plugins, 
+                  system_os, 
+                  submission_date, 
+                  system, 
+                  system_gfx, 
+                  system_cpu, 
+                  normalized_channel  
                FROM 
-                   longitudinal
+                  longitudinal
                """
 
-    spark = (SparkSession
-         .builder
-         .appName("hardware_report_dashboard")
-         .getOrCreate())
-    sqlContext = SQLContext(spark.sparkContext)
-
-    frame = sqlContext.sql(sqlQuery).where("normalized_channel = 'release'").where(
-        "build is not null and build[0].application_name = 'Firefox'")
+    frame = spark.sql(sqlQuery).where("normalized_channel = 'release'").where("build is not null and build[0].application_name = 'Firefox'")
```

Comment:
This line is too long. Is the flake8 checking working in the build? Weird that it didn't catch this. Maybe run all of this through flake8 manually, with max-line-length=100.